Gruntfile.js

```diff
@@ -360,6 +360,7 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-contrib-clean');
     grunt.loadNpmTasks('grunt-contrib-copy');
     grunt.loadNpmTasks('grunt-contrib-compress');
+    grunt.loadNpmTasks('grunt-contrib-uglify');
```

Comment:
Does this change belong in a commit for, "Developer documentation readme"?

Reply:
removing